app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.codesoom.assignment;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URI;
+
+public class DemoHttpHandler implements HttpHandler {
```

Comment:
코드를 읽는 사람을 위해 클래스 주석을 남겨 보시면 어떨까요? JavaDoc 클래스 주석은 다음과 같은 형식으로 남깁니다.

```suggestion
/**
 * (이 클래스가 하는 일)
 */
public class DemoHttpHandler implements HttpHandler {
```

Reply:
작성해봤는데 잘 한건지 모르겠네요 ㅠㅠ
다른 라이브러리나 다른 사람들은 어떻게 java doc을 작성하는지 볼 수 있는 기회가 될 것 같아요.
감사합니다!

Reply:
JavaDoc을 공부할 때 제일 좋은 자료는 Java 기본 클래스들입니다. 예를 들어 `String.java` 같은 것입니다.